src/site/content/en/blog/file-system-access/index.md

```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
   calling `close()` or when the stream is automatically closed by the pipe.
 {% endAside %}
 
+### Specifying a suggested file name and start directory
+
+In many cases your application may already have an opinion when it comes to default file names.
```

Comment:
```suggestion
In many cases you may want your app to suggest a default file name or location.
```

Reply:
Two reasons:

* Opening sentence should cover both.
* Anthropomorphizing computers is a tech writing anti-pattern.